validators/user/user.validation.js

```diff
@@ -0,0 +1,19 @@
+const { user } = require("./user.validation.schema");
+
+module.exports = {
+  addUserValidation: async (req, res, next) => {
+    const value = await user.validate(req.body);
+    if (value.error) {
+      res.status(400).json({
+        code: 400,
+        success: false,
```

Comment:
again this should be a separate func, which should be exported in module.exports.

Reply:
okay sir